src/megaclient.cpp

```diff
@@ -1049,7 +1049,7 @@ void MegaClient::init()
     mNotifiedSumSize = 0;
 
     mBizMode = BIZ_MODE_UNKNOWN;
-    mBizStatus = BIZ_STATUS_INACTIVE;
+    mBizStatus = BIZ_STATUS_UNKNOWN;
```

Comment:
If the `BIZ_STATUS_UNKNOWN` will be exposed to apps, it needs to be added to the documentation in `MegaApi`

Reply:
Fixed